internal/pkg/mount/switchroot/switchroot.go

```diff
@@ -104,6 +119,40 @@ func Switch(prefix string, mountpoints mount.Points) (err error) {
 		return fmt.Errorf("error executing /sbin/init: %w", err)
 	}
 
+	runtime.UnlockOSThread()
+
+	return nil
+}
+
+func initSelinux(selinuxMode string) error {
+	if selinuxMode == "disabled" {
+		log.Println("SELinux disabled")
+
+		return nil
+	}
+
+	log.Println("loading SELinux policy")
+
+	var err error
+
+	if err = os.WriteFile("/selinux/load", selinuxPolicy, 0o777); err != nil {
+		return err
+	}
+
+	log.Println("SELinux policy loaded")
+
+	if err = os.WriteFile("/proc/thread-self/attr/exec", []byte("system_u:system_r:init_t:s0"), 0o777); err != nil {
+		return err
+	}
+
+	if selinuxMode == "enforcing" {
+		log.Println("Setting SELinux mode to enforcing")
```

Comment:
nit: use lowercase logs
```suggestion
		log.Println("setting SELinux mode to enforcing")
```